src/Cake.XamarinStudio/Properties/Manifest.addin.xml

```diff
@@ -5,22 +5,32 @@
 			<File pattern="*.cake" />
 		</MimeType>
 	</Extension>
-	<Extension path="/MonoDevelop/SourceEditor2/SyntaxModes">
+    
+    <!-- Old way to declare a hightlight syntax
+    <Extension path="/MonoDevelop/SourceEditor2/SyntaxModes">
 		<Templates resource="Cake.XamarinStudio.Syntax.CakeSyntaxMode.xml" />
-	</Extension>
+	</Extension>-->
+    
+    <Extension path="/MonoDevelop/Ide/Editor/Bundles">       
+        <Bundle resource = "Cake.XamarinStudio.Syntax.cake.sublime-syntax" />
```

Comment:
Since 6.x also doesn't understand Sublime syntaxes, this completely breaks syntax highlighting for scripts in 6.x. Still have to test that this works with 7.x